packages/itwinui-react/src/core/Checkbox/Checkbox.tsx

```diff
@@ -104,6 +109,12 @@ export const Checkbox = React.forwardRef((props, ref) => {
     </>
   );
 
+  const {
+    className: wrapperClassName,
+    style: wrapperStyle,
```

Comment:
Since we're not doing any manipulation to `wrapperStyle` before we add it to the wrapper element, this doesn't need to be extracted. `style` customization will be included when `...restWrapperProps` is added to the wrapper element